testing/config.go

```diff
@@ -19,26 +19,42 @@ type TendermintConfig struct {
 	TrustingPeriod  time.Duration
 	UnbondingPeriod time.Duration
 	MaxClockDrift   time.Duration
-	IsWasmClient    bool
 }
 
-func NewTendermintConfig(isWasmClient bool) *TendermintConfig {
+func NewTendermintConfig() *TendermintConfig {
 	return &TendermintConfig{
 		TrustLevel:      DefaultTrustLevel,
 		TrustingPeriod:  TrustingPeriod,
 		UnbondingPeriod: UnbondingPeriod,
 		MaxClockDrift:   MaxClockDrift,
-		IsWasmClient:    isWasmClient,
 	}
 }
 
 func (tmcfg *TendermintConfig) GetClientType() string {
-	if tmcfg.IsWasmClient {
-		return exported.Wasm
-	}
 	return exported.Tendermint
 }
 
+type WasmTendermintConfig struct {
+	TendermintConfig
+	CodeHash []byte
+}
+
+func NewWasmTendermintConfig(codeHash []byte) *WasmTendermintConfig {
+	return &WasmTendermintConfig{
+		CodeHash: codeHash,
+		TendermintConfig: TendermintConfig{
+			TrustLevel:      DefaultTrustLevel,
+			TrustingPeriod:  TrustingPeriod,
+			UnbondingPeriod: UnbondingPeriod,
+			MaxClockDrift:   MaxClockDrift,
+		},
```

Comment:
maybe add a godoc saying that this is assuming the same values are used within the contract code? we could add extra godoc to the struct type definition with some basic info. 

Are we also able to use `NewTendermintConfig` or no? because it returns a pointer as opposed to value?

Reply:
you should still be able to deref it